DirectX11/HackerDXGI.cpp

```diff
@@ -167,6 +167,104 @@ HackerDXGISwapChain::HackerDXGISwapChain(IDXGISwapChain *pSwapChain, HackerDevic
 	}
 }
 
+HackerDXGISwapChain::HackerDXGISwapChain(IDXGISwapChain *pSwapChain, HackerDevice *pDevice, HackerContext *pContext, const DXGI_SWAP_CHAIN_DESC* fake_swap_chain_desc, UINT new_width, UINT new_height)
```

Comment:
Sure, you are right. That would looks much better. I guess it should work, we only need to declare all relevant hackerswapchain members as virtual. Ill take a look at it as soon as i have the access to my pc.